policies/glossary.md

```diff
@@ -83,6 +89,11 @@ source code repository.
 End-user documentation is documentation intended for users of the OpenSSL
 libraries and commandline utilities.
 
+## Functional behaviour
```

Comment:
Question: is there a policy somewhere that British spelling is preferred? I've had a look but can't find one - apologies if I'm missing something obvious. (Note: I am from the UK, so this would be my natural preference, the question is specifically about a policy to avoid confusion)

Reply:
We have tended to use British spelling - but I don't think its a formal written down policy anywhere. I'm sure there are plenty of examples of US spelling in the docs etc. Probably there is a mix.

Reply:
We have noted that British spelling is used on occasion.  I'm not sure if it ever made it into a policy document or not.  I can't easily find it.  As @mattcaswell noted, we've got a mix.

Reply:
I've added a bit about using British English and following the LDP projects guidelines.

Reply:
> I've added a bit about using British English

Did you? I don't see it.

Reply:
It's in the documentation policy: https://github.com/openssl/technical-policies/blob/8385661d131113732f6feb714d4e78a5f7728489/policies/documentation-policy.md#chapter-11-language

Reply:
At least the LDP link doesn't work

Reply:
But it's added here...

Reply:
@kroeckx The `[documentation policy]` link points to `https://github.com/openssl/technical-policies/blob/master/policies/documentation-policy.md#language`, where the fragment `#language` is added by the open PR https://github.com/openssl/technical-policies/pull/27/files

So when both PRs are merged the link should work - or have I misunderstood your question?

Reply:
It should work once both are merged.  It's a problem having two policy repositories and only one glossary.

Reply:
Re the LDP, it remains agnostic regarding British vs American English.  See for yourselves (from https://tldp.org/HOWTO/html_single/LDP-Reviewer-HOWTO/#languagereview):

> **Spelling.**  Spelling should conform to a standardized English spelling of terms. For words that are new to the language and not yet standardized (for example technical Linux terminology that is generally accepted in the community), follow the most common spelling for the term.
>
> > **Note**
> > Because there are two generally accepted forms of English, this review should not privilege American English spellings over British English spellings, or vice-versa. For example, if the author is writes British English and uses the word "realise" you should not change the spelling of the word to "realize" just because you speak/write American English.

Reply:
Sorry, this is **_way_** too late for this to impact this vote.  I do agree but I did announce this PR well back with the intention of gathering comments like this well before calling the vote.

It feels like it was a complete waste of effort...

Reply:
It was just an informative note re the discussion being held here.  What you added to this file looks fine to me.